perun-core/src/main/java/cz/metacentrum/perun/core/impl/PerunPoliciesContainer.java

```diff
@@ -1,25 +1,59 @@
 package cz.metacentrum.perun.core.impl;
 
-import com.fasterxml.jackson.databind.JsonNode;
+import cz.metacentrum.perun.core.api.PerunPolicy;
 import cz.metacentrum.perun.core.api.exceptions.PolicyNotExistsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
 import java.util.Map;
+import java.util.Queue;
 
 /**
- * PerunPoliciesContainer stores policies in a HashMap.
- * Key is a name of the policy and value is JsonNode which represent the particular policy.
+ * PerunPoliciesContainer stores a list of perun policies.
  */
 public class PerunPoliciesContainer {
 
-	private Map<String, JsonNode> perunPolicies = new HashMap<>();
+	private static final Logger log = LoggerFactory.getLogger(PerunBasicDataSource.class);
```

Comment:
Shouldn't it be like this?
```suggestion
	private static final Logger log = LoggerFactory.getLogger(PerunPoliciesContainer.class);
```

Reply:
logger not used. Erased